### PR TITLE
New Icon: SQL Developer

### LINKED ISSUE #8221: Request: SQL Developer
### Brand Name

SQL Developer

### Website

https://www.oracle.com/au/database/sqldeveloper/

### Popularity Metric

The SimilarWeb.com rank is 1,603. See: https://www.similarweb.com/website/oracle.com

### Official Resources for Icon and Color

SVG: https://www.oracle.com/a/ocom/img/sql-dev3.svg
Colour: `#ADADAD`

### Additional Comments

The Oracle icon  refers to the Company, while the Oracle SQL Developer icon  refers to the IDE used to write SQL script for Oracle databases. The Oracle icon is already available, while but the Oracle SQL Developer icon is not; hence why raise this issue.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by chrimaho**
oraclesqldeveloper

**Issue:** closes #8221

**Similarweb rank:** 1,603
https://www.similarweb.com/website/oracle.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
The Oracle icon  refers to the Company, while the Oracle SQL Developer icon  refers to the IDE used to write SQL script for Oracle databases. The Oracle icon is already available, while but the Oracle SQL Developer icon is not; hence why raise this PR.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8115,6 +8115,17 @@
             "source": "https://www.oracle.com/opn/index.html",
             "guidelines": "https://www.oracle.com/legal/logos.html"
         },
+        {
+            "title": "Oracle SQL Developer",
+            "hex": "ADADAD",
+            "source": "https://www.oracle.com/au/database/sqldeveloper",
```
Comment (adamrusted) [inline _data/simple-icons.json:8121]:
```suggestion
            "source": "https://www.oracle.com/database/sqldeveloper",
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8115,6 +8115,17 @@
...
+            "guidelines": "https://www.oracle.com/legal/logos.html",
+            "aliases": {
+                "aka": [
+                    "SQL Developer"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:8127]:
```suggestion
            "guidelines": "https://www.oracle.com/legal/logos.html"
```

File: icons/oraclesqldeveloper.svg
Comment (adamrusted) [inline icons/oraclesqldeveloper.svg:1]: I'm not 100% on the tiny gap at the top of the stack. Would you be able to expand that to 1px, instead of 0.5?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by chrimaho** [comment]
Now fixed 👍 
Thanks!

**Comment by chrimaho** [comment]
image

File: icons/oraclesqldeveloper.svg
Comment (adamrusted) [inline icons/oraclesqldeveloper.svg:1]: image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Hey @chrimaho - I'm still seeing some differences in the icon SVG unfortunately.
Would you be up for giving this another go, or would you me to lend a hand?

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @chrimaho?

**Comment by chrimaho** [comment]
Yes. Please give me some more time.

**Comment by PeterShaggyNoble** [comment]
Closing due to inactivity. If you would like to continue working on this, @chrimaho then we can reopen it for you. If someone else would like to take it on then please open a new PR.

**Pull request closed without merging**